lib/ex_phone_number/metadata/phone_metadata.ex

```diff
@@ -335,14 +358,14 @@ defmodule ExPhoneNumber.Metadata.PhoneMetadata do
       end
     %PhoneNumberDescription{
       national_number_pattern: national_number_pattern,
-      possible_number_pattern: possible_number_pattern,
+      possible_lengths: possible_lengths,
```

Comment:
If moving the possible lengths logic for general possible lengths is indeed moved to validation module, then just use `description.possible_lengths` here.